docs/appendices/release-notes/5.10.4.rst

```diff
@@ -47,3 +47,6 @@ Fixes
 - Fixed NPE when querying the :ref:`sys.allocations <sys-allocations>` table
   while no master node has been discovered. A proper exception is now thrown
   instead of an NPE.
+
+- Fixed an issue that caused queries submitted via PG wire that failed also
+  failed to be logged in :ref:`sys.jobs_log <sys-logs>`.
```

Comment:
Looks like a type with double usage of "failed" and this is specific only to parsing failures

```suggestion

- Fixed an issue that caused queries submitted via PG wire to 
not be logged in :ref:`sys.jobs_log <sys-logs>` in case of a parsing failure.
```

We need a 5.9 entry because we already have a fix in 5.9.13